### PR TITLE
NOISSUE-conveyor-improvements

### PR DESCRIPTION
**Pull request by cyraxred**
**- What I did**
- Added PulseStartedAt() on PulseSlot.
- Exposed pulse.Range value on PulseSlot.
- Added conveyor.AddInputExt for additional features.

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: ledger-core/v2/conveyor/conveyor.go [hunk 1]
```diff
@@ -118,16 +118,26 @@ func (p *PulseConveyor) GetPublishedGlobalAliasAndBargeIn(key interface{}) (smac
...
 	case err != nil:
 		return err
 	case pulseSlotMachine == nil || pulseState == 0:
 		return fmt.Errorf("slotMachine is missing: pn=%v", pn)
 	}
+	if prevPulse {
+		pr, _ := pulseSlotMachine.pulseSlot.pulseData.PulseRange()
+		return p.AddInputExt(ctx, pr.LeftBoundNumber(), event, false, createDefaults)
+	}
+	// pulseSlotMachine.pulseSlot.prevPulse
```
Comment (ruz) [inline ledger-core/v2/conveyor/conveyor.go:138]: this comment makes no sense, should be deleted

File: ledger-core/v2/conveyor/conveyor.go [hunk 2]
```diff
@@ -335,8 +345,8 @@ func (p *PulseConveyor) PreparePulseChange(out PreparePulseChangeChannel) error
 		if !ctx.CallDirectBargeIn(p.presentMachine.SlotLink().GetAnyStepLink(), func(ctx smachine.BargeInContext) smachine.StateUpdate {
 			return p.presentMachine.preparePulseChange(ctx, out)
 		}) {
-			//p.pdm.unsetPreparingPulse()
-			//close(out)
+			// p.pdm.unsetPreparingPulse()
+			// close(out)
```
Comment (ruz) [inline ledger-core/v2/conveyor/conveyor.go:349]: changes in commented out code :/ just drop it or don't change

File: ledger-core/v2/conveyor/conveyor.go (hunk 1 above)
Comment (cyraxred) [reply ledger-core/v2/conveyor/conveyor.go:138]: left accidentally. removed

File: ledger-core/v2/conveyor/conveyor.go (hunk 2 above)
Comment (cyraxred) [reply ledger-core/v2/conveyor/conveyor.go:349]: converted into TODO for further work

**Pull request merged**